dts/arm/st/l0/stm32l0.dtsi

```diff
@@ -264,7 +264,7 @@
 			clocks = <&rcc STM32_CLOCK_BUS_APB2 0x00000200>;
 			interrupts = <12 0>;
 			status = "disabled";
-			vref-mv = <3000>;
+			vref-mv = <3300>;
```

Comment:
> Actually, the vref is a board value rather than a soc one

Then maybe better to define it in the board dts rather than the SoC dts?

Reply:
I agree this should is board stuff. Problem is that this is a tree wide change and many boards would be impacted, which I'm not sure sure be done at this step. Also, this might require to be handled differently (as a fixed regulator, maybe ?)
I propose to keep it this way for V3.2.0 and do the global change in next DV.
@MaureenHelm Is it fine with you ?